cachito/web/content_manifest.py

```diff
@@ -0,0 +1,101 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import flask
+
+
+class ContentManifest:
+    """A content manifest associated with a Cacihto request."""
+
+    version = 1
+    json_schema_url = (
+        "https://raw.githubusercontent.com/containerbuildsystem/atomic-reactor/"
+        "f4abcfdaf8247a6b074f94fa84f3846f82d781c6/atomic_reactor/schemas/content_manifest.json"
+    )
+    unknown_layer_index = -1
+
+    def __init__(self, request=None):
+        """
+        Initialize ContentManifest.
+
+        :param Request request: the request to generate a ContentManifest for
+        """
+        self.request = request
+        self._packages = None
+        self._gopkg_data = {}
+        self._gomod_data = {}
+
+    def process_go_package(self, package, dependency):
+        """
+        Process go-package package.
+
+        :param Package package: the go-package package to process
+        :param Dependency dependency: the go-package package dependency to process
+        """
+        package = package.to_purl()
```

Comment:
Could you rename this to `package_purl`? It's harder to read when a new value is assigned to the function argument.

Reply:
Changed to purl since it stands for package URL